rendermodules/mesh_lens_correction/MeshAndSolveTransform.py

```diff
@@ -539,88 +408,154 @@ def create_A(self):
+            indices[mstep+6] = (lens_dof_start +
+                                mesh.simplices[pbary[1][:]][:, 0])
+            indices[mstep+7] = (lens_dof_start +
+                                mesh.simplices[pbary[1][:]][:, 1])
+            indices[mstep+8] = (lens_dof_start +
+                                mesh.simplices[pbary[1][:]][:, 2])
+            indices[mstep+9] = (lens_dof_start +
+                                mesh.simplices[qbary[1][:]][:, 0])
+            indices[mstep+10] = (lens_dof_start +
+                                 mesh.simplices[qbary[1][:]][:, 1])
+            indices[mstep+11] = (lens_dof_start +
+                                 mesh.simplices[qbary[1][:]][:, 2])
 
             offset += npoint_pairs*nnz_per_row
 
-        self.data = data
-        self.indices = indices
-        self.indptr = indptr
-
-        self.A = csr_matrix((data, indices, indptr))
-        self.weights = sparse.eye(weights.size, format='csr')
-        self.weights.data = weights
-        self.create_x0(self.A.shape[1], self.unique_ids.size)
-
-    def MeshAndSolve(self, render,
-                     input_stack, output_stack,
-                     match_collection, sectionId,
-                     nvertex, output_dir, outfile, default_lambda,
-                     translation_factor, lens_lambda, close_stack=True):
-        self.args = {}
-        self.args['render'] = render
-        self.args['input_stack'] = input_stack
-        self.args['output_stack'] = output_stack
-        self.args['match_collection'] = match_collection
-        self.args['sectionId'] = sectionId
-        self.args['nvertex'] = nvertex
-        self.args['output_dir'] = output_dir
-        self.args['out_html_dir'] = output_dir
-        self.args['outfile'] = outfile
-        self.args['regularization'] = {}
-        self.args['regularization']['default_lambda'] = default_lambda
-        self.args['regularization']['translation_factor'] = translation_factor
-        self.args['regularization']['lens_lambda'] = lens_lambda
-        self.args['close_stack'] = close_stack
-
-        # load the matches
-        self.load_matches()
-
-        # load tilespecs
-        self.load_tilespecs()
+        A = csr_matrix((data, indices, indptr))
+        wts = sparse.eye(weights.size, format='csr')
+        wts.data = weights
+        return A, wts, lens_dof_start
+
+    def create_transforms(self, ntiles, solution):
+        transforms = []
+        for i in range(ntiles):
+            transforms.append(renderapi.transform.AffineModel(
+                               M00=solution[0][i*3+0],
+                               M01=solution[0][i*3+1],
+                               B0=solution[0][i*3+2],
+                               M10=solution[1][i*3+0],
+                               M11=solution[1][i*3+1],
+                               B1=solution[1][i*3+2]))
+        return transforms
+
+    def MeshAndSolve(self):
```

Comment:
this is bad python style... CapitalLetterCase indicates this is a class that can be instantiated.